lib/simulation/openfisca/mapping/index.js

```diff
@@ -40,14 +40,17 @@ var applyRessources = function(src, dest, schema, dateDeValeur) {
 
     var ressourcesByType = _.groupBy(src.ressources, 'type');
     _.forEach(schema, function(definition, destKey) {
-        var srcKey, fn;
+        var srcKeys, fn;
         if (_.isString(definition)) {
-            srcKey = definition;
+            srcKeys = [definition];
+        } else if(_.isArray(definition)) {
+            srcKeys = definition;
         } else {
-            srcKey = definition.src;
+            srcKeys = [definition.src];
             fn = definition.fn;
         }
 
+    var mapRessources = function(srcKey) {
```

Comment:
Taking a deeper look at the code, splitting the function properly would mean quite a heavy refactor. We will probably have to do it at some point, but I'm afraid this PR goes out of hand if we try to do this here.

I will rebase this changeset, clean up history and remove aec37c0 and bdd3317.
Considering how much this will change the original commits, and for the sake of keeping the conversation here, I will open another PR.